README.md

```diff
@@ -1,14 +1,17 @@
-# PrivateWin10
-PrivateWin10 is an advanced Privacy tool for windows it provides a simple and comprehensive UI for tweaking privacy settings, as well as options to block system access to files and registry keys in order to enforce privacy whenever Windows does not provide a satisfying setting for that.
+# Private Win10
+Private Win10 is an advanced Privacy tool for windows it provides a simple and comprehensive UI for tweaking privacy settings, as well as options to block system access to files and registry keys in order to enforce privacy whenever Windows does not provide a satisfying setting for that.
 
-To solve privacy issues with other software the tool includes a powerful custom firewall frontend for the windows built in firewall.
+![Priv10 Dashboard](https://user-images.githubusercontent.com/727123/71313456-ff641280-2430-11ea-9836-1ebac8363030.png "Private Win10 Dashboard")
+![Priv10 Firewall Interface](https://user-images.githubusercontent.com/727123/71313455-fecb7c00-2430-11ea-915f-78f44f8b0022.png "Adjustable Private Win10 firewall interface")
+
+To solve privacy issues with other software the tool includes a powerful custom firewall frontend for the windows built-in firewall.
 
 
 # Licensing 
-This tool is open source; however the license is a bit more restrictive than typical open source licenses. If you want to publically distribute your self-compiled binaries review the LICENSE file for details.
+This tool is open source, however the license is a more restrictive than a typical open source license. If you want to publically distribute your self-compiled binaries review the LICENSE file for details.

```

Comment:
huuuu if I'm not mistaken "is a more" ??? is incorrect
"is a bit more" was fine
"is more" is fine as well
if you're looking at variations could also say "is slightly more" ....

Reply:
I forgot to remove the "a". I change it after Christmas.